packages/lisk-transaction-pool/src/job.ts

```diff
@@ -0,0 +1,22 @@
+import { Transaction } from "./transaction_pool";
+
+export class Job {
+	// tslint:disable-next-line variable-name
+	private readonly _interval: number;
+	// tslint:disable-next-line variable-name
+	private readonly _job: () => Promise<ReadonlyArray<Transaction>>;
+
+	public constructor(
+		context: object,
+		job: () => Promise<ReadonlyArray<Transaction>>,
+		interval: number,
+	) {
+		this._interval = interval;
+		this._job = job.bind(context);
+		this.start();
+	}
+
+	public start(): void {
```

Comment:
if we have start, shouldn't we be able to stop too?

Reply:
I think `job` should be cleanable in safe way, so keep the setInterval id and when stop is called, clean up?